sources/Application/Model/Project.cpp

```diff
@@ -59,6 +65,13 @@ Project::~Project() {
 	delete instrumentBank_ ;
 } ;
 
+// from: https://github.com/xiphonics/picoTracker/blob/master/sources/Application/Model/Project.cpp
```

Comment:
I'd prefer if we keep this info in the corresponding .h and .cpp, it's not necessary to have it here.